pkgs/development/python-modules/yamllint/default.nix

```diff
@@ -27,8 +27,8 @@ buildPythonPackage rec {
   ];
 
   disabledTests = [
-    # Test failure could be related to https://github.com/adrienverge/yamllint/issues/334
-    "test_find_files_recursively"
+    "test_cli"
```

Comment:
```suggestion
```
I only see one test failing.

Reply:
My apologies, the failing tests only fail on BSDs (Darwin). See [upstream issue](https://github.com/adrienverge/yamllint/issues/307).

I've modified my commit to wrap these in a `lib.optional stdenv.isDarwin` clause

Reply:
I don't think it has anything to do with that upstream issue. It's just that they assume the working directory to sort after `/etc`.

Reply:
> See [upstream issue](https://github.com/adrienverge/yamllint/issues/307).

Please add a comment linking to that issue.

Reply:
I understand, thank you.

Issue opened upstream and referenced in the build file.